scripts/pylint_extensions.py

```diff
@@ -1264,6 +1265,70 @@ def process_module(self, node):
                     'no-capital-letter-at-beginning', line=line_num + 1)
 
 
+class SpaceBelowFileOverviewChecker(checkers.BaseChecker):
+    """Checks if comments follow correct style."""
+
+    __implements__ = interfaces.IRawChecker
+    name = 'space_between_imports_and_file-overview'
+    priority = -1
+    msgs = {
+        'C0020': (
+            'No space used between file overview and import.',
+            'no space used between file overview and import',
+            'please provide a space between file overview and imports.'
```

Comment:
Line 1276 and 1277 is repeated, remove the second one.

Reply:
done

Reply:
replace _space_ with _newline_